spec/controls-spec.coffee

```diff
@@ -0,0 +1,21 @@
+Controls = require '../lib/controls'
+
+describe 'Controls', ->
```

Comment:
I'd also like to see a more integrated test of `autosave` itself that ensures it interacts with the service and doesn't save if a predicate returns true.